src/main/java/ProjectLibraryPartTwo/Entity/Author.java

```diff
@@ -0,0 +1,64 @@
+package ProjectLibraryPartTwo.Entity;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class Author {
+    private int id;
+    private String firstName;
+    private String lastName;
+
+    public Author() {
+    }
+
+    public Author(int id, String firstName, String lastName) {
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public Author createAuthor(ResultSet resultSet) {
```

Comment:
Also we write creating method  in service classes

Reply:
fixed